pom.xml

```diff
@@ -144,6 +159,14 @@
                 </executions>
                 <configuration>
                     <show>private</show>
+                    <tags>
+                        <tag>
+                            <name>todo</name>
```

Comment:
What is this?  Why is it here?  :)

Reply:
Hi,
I changed this because of the errors I was getting when building the project: Below a exemple of error because the javadoc has "@todo"

/home/jalisson_mello/NetBeansProjects/apgdiff/src/main/java/cz/startnet/utils/pgdiff/PgDiffIndexes.java:93: error: unknown tag: todo
* @todo Indexes that are depending on a removed field should not be added

Reply:
Thanks for clarifying :)